docs/accessibility.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: v2
+title: Accessibility
+bodyclass: a11y-statement-page
+---
+
+## Accessibility Statement for Leaflet
+
+This is an accessibility statement from Leaflet.
+
+<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" height="50px" viewBox="0 0 24 24" width="50px" style="fill:#199900;margin:-.75em 0 -1.5em 0;"><path d="M0 0h24v24H0z" fill="none"></path><path d="M20.5 6c-2.61.7-5.67 1-8.5 1s-5.89-.3-8.5-1L3 8c1.86.5 4 .83 6 1v13h2v-6h2v6h2V9c2-.17 4.14-.5 6-1l-.5-2zM12 6c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"></path></svg>
```

Comment:
This icon is https://fonts.google.com/icons?selected=Material+Icons&icon.query=accessibility+new. Let me know if I should remove it.

Reply:
In the Github `md-renderer` the icon is not showing, does it show on the website?

Reply:
I finally came around to install Ruby. Yes, it shows:

![a11y-statement](https://user-images.githubusercontent.com/26493779/149460674-e6b536b3-e8a6-4a0b-8989-0ea515077a22.png)

Maybe it should be a bit oversized to be a bit more impactful? Perhaps some added opacity and different positioning? 

![oversized-icon](https://user-images.githubusercontent.com/26493779/149462090-b4c64dbe-6b51-4d13-a2fa-5c0e490a93a4.png)

I don't feel strongly about it, it can be removed. But it's one of the [icons that symbolizes accessibility](https://fonts.google.com/icons?selected=Material+Icons&icon.query=accessibility), and makes the doc less dull?

Reply:
I like the second attempt. Maybe the Icon can be behinde the text. Anyway this needs to work on mobile devices too

Reply:
Improved styling a bit (IMO) to blend in more with the page in https://github.com/Leaflet/Leaflet/pull/7900/commits/d57a66c8f508a57f486a836840ab2f5db0f4db94.

![a11y-icon](https://user-images.githubusercontent.com/26493779/149639855-728bcfeb-82c9-43c4-b63f-e26969ee0e31.png)

It'd require more CSS to make it look good on mobile, also not exactly sure what that'd be, so hiding the icon on smaller screens, for now at least.